iree/samples/simple_embedding/device_local_sync.c

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A example of setting up the dylib-sync driver.
+
+#include "iree/base/api.h"
+#include "iree/hal/api.h"
+#include "iree/hal/local/executable_loader.h"
+#include "iree/hal/local/loaders/legacy_library_loader.h"
+#include "iree/hal/local/sync_driver.h"
+
+iree_status_t create_sample_device(iree_hal_device_t** device) {
+  // Set paramters for the device created in the next ste.
```

Comment:
```suggestion
  // Set parameters for the device created in the next step.
```
NIT: Those typos were unfortunately introduced by myself.

Reply:
Done.